rollup.src.config.js

```diff
@@ -0,0 +1,38 @@
+import nodeResolve from '@rollup/plugin-node-resolve';
+import sourcemaps from 'rollup-plugin-sourcemaps';
+import postcss from 'rollup-plugin-postcss';
```

Comment:
These dependencies are defined in e.g. `packages/algorithm/package.json`... but should they be under `devDependencies` in the repo's root package.json (or somewhere else)?

Reply:
let's keep them in each package as it is where they are used.